modules/benchmark/R/check_if_legal_table.R

```diff
@@ -0,0 +1,66 @@
+#' @title check_if_legal_table
+#' @details
+#' Checks if custom_table:
+#' 1. is formated correctly
+#' 2. is complete (has all of the species/pft's in both observations)
+#' 3. is condense-able (Could be represented as a hierachry)
+#' 
+#' @param observation_one a vector of plant fucntional types, or species
+#' @param observation_two anouther vector of plant fucntional types, or species
+#' @param custom_table a table that either maps two pft's to one anouther or maps custom species codes to bety id codes. 
+#' In the second case, must be passable to match_species_id. 
+#' @return \code{boolean} 
```

Comment:
Recent versions of roxygen (>=6.0, I think) let you [use markdown in roxygen blocks](https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html), as long as you enable it in the DESCRIPTION. Just FYI.